lib/logger.dart

```diff
@@ -26,7 +26,7 @@ void shareLog() async {
 
 class BreezLogger {
   BreezLogger() {
-    Logger.root.level = Level.ALL;
+    Logger.root.level = Level.INFO;
```

Comment:
@ademar111190 Think we want to log anything equal to and above CONFIG, not INFO here.